Collab Cafe page maintenance: Change HackMD to Framapad and add Invite Text (#3677)
the-turing-way/the-turing-way: book/website/community-handbook/coworking/coworking-collabcafe.md

## Attending an online Collaboration Cafe

The schedule for the collaboration cafe can be viewed from the [community calendar](https://calendar.google.com/calendar/embed?src=theturingway%40gmail.com&ctz=Europe%2FLondon).

> We also announce calls in our [monthly newsletters](https://buttondown.email/turingway/), send a reminder on the [Slack channel](https://tinyurl.com/jointuringwayslack) and on [X (formerly Twitter)](https://twitter.com/turingway) and [Mastodon](https://fosstodon.org/@turingway).

Reviewer: ```suggestion
We also announce calls in our [monthly newsletters](https://buttondown.email/turingway/), send a reminder on the [Slack channel](https://tinyurl.com/jointuringwayslack) and on [X (formerly Twitter)](https://twitter.com/turingway) and [Mastodon](https://fosstodon.org/@turingway). Our [LinkedIn account](https://www.linkedin.com/company/the-turing-way/) can also be used to announce calls.
```

Though scheduled for 2 hours, we understand that for many interested participants it might be impossible to commit to the entire 2 hours on one or multiple occasions, therefore, you are welcome to join for as long as your schedule allows.

If the current schedule of the online Collaboration Cafe is not suitable for your time zone, you are welcome to host a separate collaboration cafe for your community/time zone.
Please open a new issue (see [this example](https://github.com/the-turing-way/the-turing-way/issues/684)) or reach out to _The Turing Way_ team members for details.
Continue reading for information on hosting these calls.

### Inviting people to an online Collaboration Cafe

New attendees are welcome to join the Collaboration Cafe, and it is a space where many may encounter _The Turing Way_ project for the first time.

> Existing _The Turing Way_ community members may invite their colleagues to join them in a Pomodoro session to discuss a topic of shared interest or co-work together.

Reviewer: ```suggestion
Existing _The Turing Way_ community members may invite their colleagues to join them in a breakout session, called Pomodoro session in _The Turing Way_ community, to discuss a topic of shared interest or co-work together.
```

#### Before the Call

> - Please update the shared document ([Framapad](https://annuel2.framapad.org/p/ttw-collaboration-cafe)) before the event using the following steps:

Reviewer: ```suggestion
- Facilitators of the call, update the shared document ([Framapad](https://annuel2.framapad.org/p/ttw-collaboration-cafe)) before the event using the following steps:
```
